openapi/Dockerfile

```diff
@@ -32,6 +32,15 @@ RUN cd /source/swagger-codegen && \
     mvn install -DskipTests -Dmaven.test.skip=true -pl modules/swagger-codegen-maven-plugin -am && \
     cp -r /root/.m2/* /usr/share/maven/ref
 
+# Install Autorest
+RUN curl -sL https://deb.nodesource.com/setup_7.x | bash - 
+RUN apt-get update && apt-get -y install \
+    nodejs \
+    libunwind8-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN npm install -g autorest@preview
```

Comment:
Do we always want to install +@preview+? Perhaps we want to add a TODO here to revert to a more stable release when the features we need are there?